src/session/content/message_row/bubble.rs

```diff
@@ -193,6 +197,29 @@ impl MessageBubble {
             None
         };
 
+        // Handle MessageReply
+        if message.reply_to_message_id() != 0 {
+            let message_chat = message
+                .forward_info()
+                .and_then(|f| {
+                    if f.from_chat_id() != 0 {
+                        Some(message.chat().session().chat(f.from_chat_id()))
+                    } else {
+                        None
+                    }
+                })
```

Comment:
This seems to not work if I forward two messages in "Saved Messages":

![image](https://user-images.githubusercontent.com/23530586/223891707-03d8d671-1e86-4fe5-b281-971b73edc967.png)

The reply appear correctly in tdesktop:

![image](https://user-images.githubusercontent.com/23530586/223891789-a261fd2f-957b-48da-9af8-b385cd128a85.png)

Reply:
Maybe I was right and we need to use message id from ForwardInfo?

Reply:
I'll check if it fixes the replies and then force push

Reply:
![Screenshot from 2023-03-09 16-39-57](https://user-images.githubusercontent.com/110191114/224075470-ded3b26a-cbad-487a-91f9-430a238b43c0.png)
No, it doesn't suppose to look like that

Reply:
I think what I'm trying to achieve is overcomplicated. I think we shouldn't use forward_info and only make an exception for `@replies` chat

Reply:
Can we postpone the `@replies` handling for later then? It seems that this handling is not needed indeed for `Saved Messages`.

Reply:
sure, let's leave it for other time